static/src/javascripts-legacy/projects/common/modules/ui/tabs.js

```diff
@@ -1,88 +0,0 @@
-define([
-    'lib/$',
-    'bean',
-    'bonzo'
-], function (
-    $,
-    bean,
-    bonzo
-) {
-
-    /*
-        expects the following HTML structure
-
-        <div class="tabs">
-            <ol class="tabs__container js-tabs" role="tablist">
-                <li id="foo-tab" class="tabs__tab tabs__tab--selected" role="tab" aria-selected="true" aria-controls="foo"><a href="#foo">Foo</a></li>
-                <li id="bar-tab" class="tabs__tab" role="tab" aria-selected="false" aria-controls="bar"><a href="#bar">Bar</a></li>
-            </ol>
-
-            <div class="tabs__content">
-                 <div class="tabs__pane" id="foo" role="tabpanel" aria-labelledby="foo-tab">foo</div>
-                 <div class="tabs__pane modern-hidden" id="bar" role="tabpanel" aria-labelledby="bar-tab">bar</div>
-            </div>
-        </div>
-    */
-
-    var Tabs = function () {
-
-        var view = {
```

Comment:
This is  a tidy refactor, much easier to reason about, good to see the `showPane` and `hidePane` logic split out into separate functions, as appose this original `view.showTab` function, and good to see the needless `view` object go. I'm happy to see Bean and Bonzo go too!